sdk/scheduler/src/main/java/com/mesosphere/sdk/scheduler/DefaultScheduler.java

```diff
@@ -779,7 +779,9 @@ public void statusUpdate(SchedulerDriver driver, Protos.TaskStatus status) {
 
             // Store status, then pass status to PlanManager => Plan => Steps
             try {
-                stateStore.storeStatus(status);
+                String taskName = StateStoreUtils.getTaskName(stateStore, status);
```

Comment:
Let me re-phrase my earlier comment:

`final` is a good practice for method-local variables. We do not have any existing preference or enforcement of this. If we want to add it in as a static analysis check then we can do so.